### DISCUSSION TITLE
How to setup the package locally in my machine

### DISCUSSION THREAD

**Discussion by alsong** [Freddywhest/firestore-eloquent#10] [Ideas]
I would like to contribute to the package, but i would need guidelines of how to setup the package.
@autonomynti  or @Freddywhest  is there a documentation on how to setup, there a few features that i want to add.

**Comment by Freddywhest** [comment]
Guide on how to contribute: https://firestore-eloquent.netlify.app/docs/contribute

**Comment by Freddywhest** [comment]
Guide on how you can set up on your local machine: https://firestore-eloquent.netlify.app/docs/get-started/installation
Check the requirements

**Comment by Freddywhest** [comment]
I'm overjoyed by your desire to participate in this project

**Reply by alsong** [reply]
I just saw that this package will help alot on my current project. I just have somethings to add and i don't want to overwhelm you with features. So decided to just contribute

**Reply by Freddywhest** [reply]
That's fantastic to hear! Your enthusiasm and willingness to contribute are greatly appreciated. Adding your insights and enhancements can only make the project better. Feel free to share your ideas whenever you're ready—I'm here to collaborate and make this project the best it can be. Thank you for your initiative!

**Reply by alsong** [reply]
alright so the idea is to return the links just the way eloquent pagination returns the links. Currently you return an html element.
Now working with inertiajs i will have to render that html element. Now when i click to move to the next page i see the whole page reloading but in inertiajs its not supposed to do that. So that what i want to work on for now. I have other ideas but i want to go one at a time

**Reply by Freddywhest** [reply]
If you don't mind, I will add that and publish an update tomorrow

**Reply by alsong** [reply]
oh, is that so. Then i will be greatful. Will be looking to test it out tomorrow

**Comment by Freddywhest** [comment]
Apologies for the delay in updating the project. I've been tied up with other tasks, and unfortunately, I've also been grappling with a poor internet connection since last week.

**Reply by alsong** [reply]
its cool, i also didn't demo the project last week. Moved the demo for this week hoping the feature would be available

**Reply by alsong** [reply]
@Freddywhest do you like have a way that i can reach out to you. So that we communicate in realtime

**Reply by Freddywhest** [reply]
telegram: @roddyfred

**Comment by alsong** [comment]
@Freddywhest look at the pr https://github.com/Freddywhest/firestore-eloquent/pull/13
